changelogs/2025-03-20-changelog.html

```diff
@@ -62,17 +63,16 @@
     }
 
     section {
-      margin-top: 60px;
-      border-radius: 20px;
-      background: #fff;
-      padding: 30px;
+      margin-top: var(--pine-dimension-750);
+      border-radius: var(--pine-dimension-250);
+      background: var(--pine-color-white);
+      padding: var(--pine-dimension-400);
       box-shadow: 0 5px 5px #ececec;
-      margin-bottom: 60px;
+      margin-bottom: var(--pine-dimension-750);
     }
 
     h1 {
       margin: 0;
-      font-family: serif;
       font-size: 60px;
```

Comment:
Nitpick. Could we use a calc based on a token? 

`font-size: calc(var(--pine-font-size-heading-4) * 3);`